Sources/SwiftUINavigation/Internal/Deprecations.swift

```diff
@@ -2,6 +2,122 @@
   import SwiftUI
   @_spi(RuntimeWarn) import SwiftUINavigationCore
 
+  // NB: Deprecated after 1.3.0
+
+  @available(iOS 14, tvOS 14, watchOS 7, *)
+  @available(macOS, unavailable)
+  extension View {
+    @available(
+      *, deprecated,
+      message:
+        "Use the 'fullScreenCover(item:)' (or 'fullScreenCover(item:id:)') overload that passes a binding"
+    )
+    public func fullScreenCover<Value, Content>(
+      unwrapping value: Binding<Value?>,
+      onDismiss: (() -> Void)? = nil,
+      @ViewBuilder content: @escaping (Binding<Value>) -> Content
+    ) -> some View
+    where Content: View {
+      self.fullScreenCover(
+        isPresented: value.isPresent(),
+        onDismiss: onDismiss
+      ) {
+        Binding(unwrapping: value).map(content)
+      }
+    }
+  }
+
+  @available(iOS 16, macOS 13, tvOS 16, watchOS 9, *)
+  extension View {
+    @available(
+      *, deprecated,
+      message: "Use the 'navigationDestination(item:)' overload that passes a binding"
```

Comment:
This is not a 1-1 change, so maybe we should better message it. With this deprecation we are abandoning our attempt to work around SwiftUI bugs and instead just embrace the state of Apple's own framework code.